content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -572,6 +572,43 @@ ConfigMap updates.
 
 {{% capture discussion %}}
 
+## Automatically Created ConfigMaps
+
+{{< feature-state for_k8s_version="v1.13" state="stable" >}}
+
+If BoundServiceAccountTokenVolume was enabled, kube-controller-manager would create a ConfigMap named `kube-root-ca.crt` in each namespace.
+
+Verify that the `kube-root-ca.crt` ConfigMap exists in `default` Namespace:
+
+```shell
+kubectl get cm -n default
+```
+
+The output is similar to this:
+
+```shell 
+NAME               DATA   AGE
+kube-root-ca.crt   1      6m44s
+```
+
+This ConfigMap is created for storing ca.crt data, container would use that certificate to access api-server. You could observe it like:
```

Comment:
I wonder if it makes sense to explain this more verbosely? I'm not exactly sure what wording to propose, something like:
```suggestion
This ConfigMap stores the CA certificate for the Kubernetes cluster PKI. You could use this when a pod needs to access the apiserver: configure a container in that pod that to uses the `kube-root-ca.crt` ConfigMap, and then the app running inside the container uses that root certificate to verify the identity of the apiserver. You could observe it like this:
```
perhaps?

Reply:
I don't think we should do like this. Since this configmap would be injected  into container automatically.